script.js

```diff
@@ -14,3 +70,490 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
+      </ul>
+  </div>       
+  </div>
+  </div>
+  `;
+  homeSection.appendChild(container);
+  body.appendChild(homeSection);
+
+  portfolioContainer.innerHTML = `
+  <div class="recent-work">
+      <h3>My Recent Works</h3>
+      <hr class="recent-hr">
+  </div>
+  <div class="project-box">
+      <div class="img-holder">
+          <img id="img1" src="${workData[0].image[1]}" alt="">
+          <img id="img2" src="${workData[0].image[0]}" alt="">
+      </div>        
+      <div class="work-content">
+          <h2>${workData[0].title}</h2>
+          <p>
+              ${workData[0].description}
+          </p>
+          <div class="btn">
+              <ul>
+              <li>${workData[0].list[0]}</li>
+              <li>${workData[0].list[1]}</li>
+              <li>${workData[0].list[2]}</li>
+              <li>${workData[0].list[3]}</li>
+              </ul>
+          </div>
+          <div class="see_more_btn">
+              <button class="view-project">${workData[0].button}</button>
+          </div> 
+      </div>
+  </div>
+  <div class="work-container">
+      <div class="project-section first-em">
+        <h2>
+        ${portfolioData[2].title}
+        </h2>
+        <p>
+        ${portfolioData[2].description}
+        </p>
+        <div class="btn">
+            <ul>
+                <li>${portfolioData[2].list[0]}</li>
+                <li>${portfolioData[2].list[1]}</li>
+                <li>${portfolioData[2].list[2]}</li>
+            </ul>
+        </div>
+        <div class="see-btn">
+            <button class="view-project">${portfolioData[2].button}</button>
+        </div>
+        </div>
+        <div class="project-section first-mob">
+          <h2>
+          ${portfolioData[2].title}
+          </h2>
+          <p>
+          ${portfolioData[2].description}
+          </p>
+          <div class="btn">
+              <ul>
+              <li>${portfolioData[2].list[0]}</li>
+              <li>${portfolioData[2].list[1]}</li>
+              <li>${portfolioData[2].list[2]}</li>
+              </ul>
+          </div>
+          <div class="see-btn">
+              <button class="view-project">${portfolioData[2].button}</button>
+          </div>
+          </div>
+          <div class="project-section first-mob">
+              <h2>${portfolioData[2].title}
+              </h2>
+              <p>
+                  A daily selection of privately  <br> 
+                  personalized reads; no accounts or <br> 
+                  sign-ups required. Has been the <br> 
+                  industry's standard.
+              </p>
+              <div class="btn">
+                  <ul>
+                      <li>${portfolioData[2].list[0]}</li>
+                      <li>${portfolioData[2].list[1]}</li>
+                      <li>${portfolioData[2].list[2]}</li>
+                  </ul>
+              </div>
```

Comment:
> - [ ] Hello @Blessman-Newton,
 Based on the feedback from the previous code reviewer, it would be advisable to utilize a JavaScript array to store the information for all projects. This approach will allow for the dynamic creation of the projects section, avoiding code repetition and adhering to the _DRY_ (Don't Repeat Yourself) principle. For more insights, you can refer to the [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) article.👍